content/_layouts/frame.html.haml

```diff
@@ -87,134 +87,28 @@
     %script{:src => expand_link("/assets/bower/anchor-js/anchor.min.js")}
     %script{:src => expand_link("/assets/bower/@popperjs/core/umd/popper.min.js")}
     %script{:src => expand_link("/assets/bower/bootstrap/js/bootstrap.min.js")}
+    %script{:src => "https://unpkg.com/lit@2.4.0/polyfill-support.js"}
+    %script{:src => "https://unpkg.com/@webcomponents/webcomponentsjs@2.6.0/webcomponents-loader.js"}
```

Comment:
do we have to link this to unpkg? can't we just bundle it via npm like other assets?

Reply:
don't need to no, but unpkg is a cdn, so it'll load fast if a user has already seen it.

Reply:
as is jenkins.io and if the npm registry is used then dependency update tools can handle it out of the box.

Reply:
> as is jenkins.io and if the npm registry is used then dependency update tools can handle it out of the box.

this repo isn't using any dependency update tools, and any time javascript dependancies update, cache needs to be manually expired including any hidden sub dependencies. It made the upgrade to bootstrap 5 really difficult.

> also likely to cause less flickering I would expect and just good when developing locally, e.g. if I'm developing offline...

These two libraries are just polyfill for older browsers. Its really just needed for safarai and ie 11 at this point and i'm not even sure safari - https://caniuse.com/custom-elementsv1

For the actual jenkins component, the reason I went with "latest version on unpkg" is to make sure the toolbar is always up to date on all sites, and we don't have sites that don't get maintained having really old toolbars.

It could be version pinned and updated with updatecli but i think as long as jenkins.io doesn't have a bundler that hashes dependancies, or doesn't have a cdn that expires every deploy, we shouldn't be adding new dependancies via package.json